armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -623,6 +574,7 @@ def mapHotTempToComponents(self, tempGrid, tempField):
 
```

Comment:
This is outside the changes here, but I would appreciate a rename of this method from `mapHotTempToComponents` to `updateComponentTempsBy1DTempField` or something like that. The reason for this is that it makes it a bit more clear to me that we are supplying a 1D temperature field to the method and then that is being used to apply some average temperature to all the components in a block.

This would also allow us to expand this later to add `updateComponentTempsByComponentTempField` or something of that nature that could directly accept temperatures that we would assign to each component. The use case here is if we have a detailed thermal-hydraulic solver (or some mechanism) to get component-level temperatures it would be better to allow direct temperature assignments to the components rather than enforcing that all the components in a block have the same average temperature.

Reply:
Sounds good